examples/cifar10_to_monai.ipynb

```diff
@@ -0,0 +1,606 @@
+    "        100 * correct / total))\n",
+    "    \n",
+    "get_accuracy(testloader, net, device)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Use transforms from MONAI"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we will see how easy it is to use parts from MONAI to augment your existing code. Let's begin with using some transforms from MONAI.\n",
+    "\n",
+    "First, We will reset the network parameters so we can train it again."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 65,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "Net(\n",
+       "  (conv1): Conv2d(3, 6, kernel_size=(5, 5), stride=(1, 1))\n",
+       "  (pool): MaxPool2d(kernel_size=2, stride=2, padding=0, dilation=1, ceil_mode=False)\n",
+       "  (conv2): Conv2d(6, 16, kernel_size=(5, 5), stride=(1, 1))\n",
+       "  (fc1): Linear(in_features=400, out_features=120, bias=True)\n",
+       "  (fc2): Linear(in_features=120, out_features=84, bias=True)\n",
+       "  (fc3): Linear(in_features=84, out_features=10, bias=True)\n",
+       ")"
+      ]
+     },
+     "execution_count": 65,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "def weight_reset(m):\n",
+    "    if isinstance(m, nn.Conv2d) or isinstance(m, nn.Linear):\n",
+    "        m.reset_parameters()\n",
+    "        \n",
+    "net.apply(weight_reset)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next, we will write a simple transform called `PILToNumpy` to convert the input from CIFAR-10 dataset to numpy arrays because MONAI transforms work on numpy arrays. Then, we can simple add the `AsChannelFirst`, `RandFlip`, `RandRotate` and `IntensityNormalizer` transforms. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 66,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class PILToNumpy(object):\n",
+    "    def __init__(self):\n",
+    "        pass\n",
+    "    \n",
+    "    def __call__(self, img):\n",
+    "        assert isinstance(img, Image.Image), \"input must be a PIL image.\"\n",
+    "\n",
+    "        return np.array(img).astype(np.float32)\n",
+    "\n",
+    "transform = transforms.Compose(\n",
+    "                [PILToNumpy(),\n",
+    "                 monai.transforms.AsChannelFirst(),\n",
+    "                 monai.transforms.RandFlip(prob=0.5, spatial_axis=[1, 0]),\n",
+    "                 monai.transforms.RandRotate(prob=0.5, degrees=90),\n",
+    "                 monai.transforms.IntensityNormalizer(np.array((0, 0, 0)), \n",
+    "                                                      np.array((255, 255, 255))),\n",
+    "                 monai.transforms.GaussianNoise(),\n",
+    "                 monai.transforms.ToTensor(),\n",
+    "                 transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))])"
```

Comment:
Why you add **Normalize** again as you already added "IntensityNormalizer"? I think they are the same logic.

Reply:
IntensityNormalizer works on images and Normalize on tensors. One is to get image to 0-1 range. Other is to convert output to -0.5-0.5 range. Explained this.

Reply:
OK, actually you can set (0.5, 0.5, 0.5) to `IntensityNormalizer` and no need to call `Normalize`.
But it's better to show how we compose with torchvision transforms together, so let's keep it.
`IntensityNormalizer` had been changed name to `NormalizeIntensity` in the latest code, please merge master and update your tutorial.